src/main.rs

```diff
@@ -82,10 +82,10 @@ fn try_main() -> Result<(), Box<dyn std::error::Error>> {
 
     match solve_result.status {
         SolveStatus::TimedOut { elapsed } => {
-            eprintln!("⚠️  Timed out after {:.1}s; only partial solutions returned", elapsed.as_secs_f64());
+            eprintln!("⚠️  Timed out after {:.1}s; some solutions may not have been returned", elapsed.as_secs_f64());
         }
         SolveStatus::FoundEnough => {
-            eprintln!("✓ Stopped after finding {} solutions", solve_result.solutions.len());
+            eprintln!("✓ Stopped after finding {}/{} requested solutions", solve_result.solutions.len(), cli.num_results_requested);
```

Comment:
(not a rhetorical question) will it be weird to (potentially) see a fraction that's greater than 1?

Reply:
I don't think so (but also I don't expect that to ever happen)